src/Cache/Engine/FileEngine.php

```diff
@@ -103,17 +103,6 @@ public function init(array $config = [])
     }
 
     /**
-     * Garbage collection. Permanently remove all expired and deleted data
-     *
-     * @param int|null $expires [optional] An expires timestamp, invalidating all data before.
-     * @return bool True if garbage collection was successful, false on failure
-     */
-    public function gc($expires = null)
```

Comment:
If we're removing this method, we should also consider removing `Cache::gc()`, and the call to gc() when engines are created by CacheRegistry.